### PR TITLE
Windows installer: Windows installer not overwriting libiio

### PR DESCRIPTION
**Pull request by AlexandraTrifan**
The Windows installer was not overwriting libiio because the file version both for a new and old version was 0.0.0.0. We added the "ignoreversion" flag which will allow the installer to always install the selected version.
Also, a file version was added to the installed files, so from now on it will be 0.19, 0.20,..

### CODE REVIEW

File: libiio.iss.cmakein [hunk 1]
```diff
@@ -44,11 +44,11 @@ Name: "turkish"; MessagesFile: "compiler:Languages\Turkish.isl"
...
-Source: "C:\projects\libiio\build-win64\Release\libiio.dll"; DestDir: "{sys}"; Check: Is64BitInstallMode
+Source: "C:\projects\libiio\build-win32\Release\libiio.dll"; DestDir: "{sys}"; Flags: 32bit ignoreversion
+Source: "C:\projects\libiio\build-win64\Release\libiio.dll"; DestDir: "{sys}"; Check: Is64BitInstallMode; Flags: ignoreversion
 
-Source: "C:\projects\libiio\build-win32\tests\Release\*.exe"; DestDir: "{sys}"; Check: not Is64BitInstallMode
-Source: "C:\projects\libiio\build-win64\tests\Release\*.exe"; DestDir: "{sys}"; Check: Is64BitInstallMode
+Source: "C:\projects\libiio\build-win32\tests\Release\*.exe"; DestDir: "{sys}"; Check: not Is64BitInstallMode; Flags: ignoreversion
+Source: "C:\projects\libiio\build-win64\tests\Release\*.exe"; DestDir: "{sys}"; Check: Is64BitInstallMode; Flags: ignoreversion
 
 Source: "C:\projects\libiio\build-win32\Release\libiio.lib"; DestDir: "{pf32}\Microsoft Visual Studio 12.0\VC\lib"; Check: not Is64BitInstallMode
```
Comment (dNechita) [inline libiio.iss.cmakein:53]: Should these get the 'ignoreversion' flag as well?

Comment (AlexandraTrifan) [reply libiio.iss.cmakein:53]: If they don't have the flag, they will not be copied in the system. This means that we could install a new libiio, use the latest library version, but not that latest iio_info for example.

Comment (AlexandraTrifan) [reply libiio.iss.cmakein:53]: Sorry, got this question wrong. I checked the logs and, unlike .exe and.dll, the .lib file does not have a file version at all. This means that the installer will not take into consideration any version, will check the timestamps and copy the provided file.

This is a snapshot from the log file:
` Dest filename: C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\lib\amd64\libiio.lib`
` Time stamp of our file: 2020-05-28 07:02:02.000`
`  Dest file exists.`
` Time stamp of existing file: 2019-01-10 15:19:26.000`
` Version of our file: (none)`
` Version of existing file: (none)`
` Installing the file.`

**Comment by rgetz** [comment]
From the build:

Warning: Unsafe flag usage on file "libiio.dll": The "ignoreversion" flag should not be used on files installed to the Windows System directory ("{sys}").
2033Warning: Unsafe flag usage on file "libiio.dll": The "ignoreversion" flag should not be used on files installed to the Windows System directory ("{sys}").
2034Warning: Unsafe flag usage on file "iio_attr.exe": The "ignoreversion" flag should not be used on files installed to the Windows System directory ("{sys}").
2035Warning: Unsafe flag usage on file "iio_genxml.exe": The "ignoreversion" flag should not be used on files installed to the Windows System directory ("{sys}").
2036Warning: Unsafe flag usage on file "iio_info.exe": The "ignoreversion" flag should not be used on files installed to the Windows System directory ("{sys}").
2037Warning: Unsafe flag usage on file "iio_readdev.exe": The "ignoreversion" flag should not be used on files installed to the Windows System directory ("{sys}").
2038Warning: Unsafe flag usage on file "iio_reg.exe": The "ignoreversion" flag should not be used on files installed to the Windows System directory ("{sys}").
2039Warning: Unsafe flag usage on file "iio_writedev.exe": The "ignoreversion" flag should not be used on files installed to the Windows System directory ("{sys}").
2040Warning: Unsafe flag usage on file "iio_attr.exe": The "ignoreversion" flag should not be used on files installed to the Windows System directory ("{sys}").
2041Warning: Unsafe flag usage on file "iio_genxml.exe": The "ignoreversion" flag should not be used on files installed to the Windows System directory ("{sys}").
2042Warning: Unsafe flag usage on file "iio_info.exe": The "ignoreversion" flag should not be used on files installed to the Windows System directory ("{sys}").
2043Warning: Unsafe flag usage on file "iio_readdev.exe": The "ignoreversion" flag should not be used on files installed to the Windows System directory ("{sys}").
2044Warning: Unsafe flag usage on file "iio_reg.exe": The "ignoreversion" flag should not be used on files installed to the Windows System directory ("{sys}").
2045Warning: Unsafe flag usage on file "iio_writedev.exe": The "ignoreversion" flag should not be used on files installed to the Windows System directory ("{sys}").


From the [doc](https://jrsoftware.org/ishelp/index.php?topic=filessection)
```
ignoreversion
Don't compare version info at all; replace existing files regardless of their version number.

This flag should only be used on files private to your application, never on shared system files
```

**Pull request merged**